test/lib/ansible_test/_internal/commands/sanity/__init__.py

```diff
@@ -123,6 +123,11 @@
 SANITY_ROOT = os.path.join(ANSIBLE_TEST_CONTROLLER_ROOT, 'sanity')
 TARGET_SANITY_ROOT = os.path.join(ANSIBLE_TEST_TARGET_ROOT, 'sanity')
 
+# NOTE: must match ansible.constants.DOCUMENTABLE_PLUGINS, but with 'module' replaced by 'modules'!
+DOCUMENTABLE_PLUGINS = (
+    'become', 'cache', 'callback', 'cliconf', 'connection', 'httpapi', 'inventory', 'lookup', 'netconf', 'shell', 'vars', 'modules', 'strategy'
```

Comment:
It would be good to keep the plugin list sorted:

```suggestion
    'become', 'cache', 'callback', 'cliconf', 'connection', 'httpapi', 'inventory', 'lookup', 'modules', 'netconf', 'shell', 'strategy', 'vars'
```

Reply:
My goal was to keep the list as close to the one in `ansible.constants`. But I can also sort it.

Reply:
Addressed in accfc7f632680fefa41206deb040e67b2d879400.